### PR TITLE
Fw 5019 form fields clarification

### PR DESCRIPTION
**Pull request by Cara-Barter**
### Description of Changes

corrected casing and wording in the dashboard for all forms, etc.

### Checklist

- [ ] README / documentation has been updated if needed
- [x] PR title / commit messages contain Jira ticket number if applicable
- [ ] Tests have been updated / created if applicable

### Reviewers Should Do The Following:

- [x] Review the changes here
- [ ] Pull the branch and test locally

### Additional Notes

N/A

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -33,6 +33,8 @@ module.exports = {
     'jsx-a11y/alt-text': ['warn'],
     'jsx-a11y/aria-role': ['warn'],
     'jsx-a11y/media-has-caption': ['off'],
+    'cypress/no-unnecessary-waiting': ['off'],
+    'cypress/unsafe-to-chain-command': ['warn'],
```
Comment (sahil97) [inline .eslintrc.js:37]: I do not think we should disable/downgrade the error level of these on a project level. If there is a specific place this cannot be fixed, we should add a comment for es-lint to ignore it there, but our priority should be to fix these.
Adding waiting time to test can introduce unexpected behaviour in tests.

File: src/components/Form/CategoryArrayField.js [hunk 2]
```diff
@@ -56,16 +56,8 @@ function CategoryArrayField({
...
         {fields?.length < maxItems && (
-          <button
-            type="button"
-            onClick={() => openModal()}
-            className="bg-white border-2 border-primary text-primary hover:bg-gray-50 rounded-lg shadow-sm py-2 px-4 inline-flex justify-center text-sm font-medium  focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-secondary-light"
-          >
-            {getIcon('Add', 'fill-current -ml-1 mr-2 h-5 w-5')}
-            <span>Add {label}</span>
-          </button>
+          <FieldButton label={label} openModal={openModal} />
```
Comment (sahil97) [inline src/components/Form/CategoryArrayField.js:60]: +1 for moving the button into a re-usable component.

**Review by sahil97 — CHANGES_REQUESTED** [review]
Looks good. Other than eslintrc changes, looks good to me.

File: .eslintrc.js (hunk 1 above)
Comment (Cara-Barter) [reply .eslintrc.js:37]: I spoke with Chris about this and he wanted this added in.

Comment (madebybridget) [reply .eslintrc.js:37]: @chrisfpcc We shouldn't disable linting in order to preserve test reliability. @Cara-Barter Please remove, thanks!

**Review by sahil97 — APPROVED** [review]

**Pull request merged**